index.bs

```diff
@@ -191,8 +206,7 @@ below and in [[#index-defined-elsewhere]].
 
 : HTML
 :: The concepts of [=relevant settings object=], [=origin=],
-    [=opaque origin=], [=is a registrable domain suffix of or is equal to=], and the {{Navigator}} interface are
-    defined in [[!HTML52]].
+    [=opaque origin=], and [=is a registrable domain suffix of or is equal to=] are defined in [[!HTML52]].
 
 : Web Cryptography API
 :: The {{AlgorithmIdentifier}} type and the method for [=normalizing an algorithm=] are defined in
```

Comment:
Added in https://github.com/battre/webauthn/commit/f49e915768636bae1d31fd5d74fe8c7cffff1b6b.